AppCenterDataStorage/AppCenterDataStorage/Internal/MSDataStoreInternal.h

```diff
@@ -18,6 +20,12 @@ NS_ASSUME_NONNULL_BEGIN
 @property(nonatomic, copy) NSURL *tokenExchangeUrl;
 
 /**
+ * A local store instance that is used to manage application and user level documents.
+ */
+@property(nonatomic, nullable) id<MSDocumentStore> documentStore;
```

Comment:
It is instantiated in `init`. Doesn't seem `nullable`.
```suggestion
@property(nonatomic) id<MSDocumentStore> documentStore;
```